templates/medium/cxf-contract-first/medium-fuse-cxf-contract-first-java/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,5 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: [B@183d80d
```

Comment:
Please remove those Manifest. They should be generated on build time.
Am 07.07.2016 7:53 nachm. schrieb "Aurélien Pupier" <
notifications@github.com>:

> In
> templates/medium/cxf-contract-first/medium-fuse-cxf-contract-first-java/src/main/resources/META-INF/MANIFEST.MF
> <https://github.com/fusesource/fuseide/pull/574#discussion_r69953999>:
>
> > @@ -0,0 +1,5 @@
> > +Manifest-Version: 1.0
> > +Bundle-ManifestVersion: 2
> > +Bundle-Name: [B@183d80d
>
> strange value
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/fusesource/fuseide/pull/574/files/ba2e971eebe7cc269094abfbf89460d08ac6a5dc#r69953999>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAamjT8AzT1FPl71b3Uc_OJle-IF7g6fks5qTT0ZgaJpZM4JHVY8>
> .
>